edx_theme/__init__.py

```diff
@@ -47,6 +47,14 @@ def setup(app):
 
     Arguments:
         app (Sphinx): Application object for the Sphinx process
+
+    Returns:
+        a dictionary of metadata (http://www.sphinx-doc.org/en/stable/extdev/#extension-metadata)
     """
     event = 'html-page-context' if six.PY3 else b'html-page-context'
     app.connect(event, update_context)
+
+    return {
+        'parallel_read_safe': True,
+        'parallel_write_safe': True,
```

Comment:
We should probably add "version" to this dictionary while we're at it, using `__version__` from above. http://www.sphinx-doc.org/en/latest/extdev/index.html#extension-metadata